src/component/vocabulary/CreateVocabulary.tsx

```diff
@@ -148,8 +144,8 @@ export class CreateVocabulary extends AbstractCreateAsset<CreateVocabularyProps,
 
                             <Files
                                 files={this.state.files}
-                                createFile={onCreateFile}
-                                onFileAdded={this.onFileAdded}
+                                createFile={this.onCreateFile}
+                                onFileAdded={() => {;}}
```

Comment:
I think it is much clearer to have the onFileAdded as an optional prop in Files, or have a default prop for it (which would be an empty function).